src/app/chef/[id]/page.tsx

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/first */
+
+/* eslint-disable tailwindcss/classnames-order */
+/* eslint-disable tailwindcss/no-custom-classname */
+
+import Link from "next/link";
+
+import { mockDataRecipe, RecipeAppTeam10 } from "@/mock";
+
+import { FollowButton } from "@/components/button";
+import { ArrowIcon, ThreeDotsIcon } from "@/components/icons";
+import { ImageComponent, ImageGrid } from "@/components/image";
+
+/* eslint-disable import/first */
+const recipeNumber = 1234;
+const followerNumber = 5678;
+
+export const generateStaticParams = () => {
+  return [{ id: "1" }, { id: "2" }, { id: "3" }];
+};
+// https://nextjs.org/docs/app/api-reference/functions/generate-static-params
+// 上記参考に動的ルーティングに対応しておりますが、DBと合わせた際の挙動まで考えて実装していません。
+
+const ChefPage = ({ params }: { params: { id: string } }) => {
+  const { id } = params;
+  return (
+    <div className="relative mx-auto">
+      <button type="button" className="absolute left-3 top-3 z-10 cursor-pointer rounded-full p-1.5">
+        <Link href={{ pathname: `/` }} passHref>
+          <ArrowIcon />
+        </Link>
+      </button>
+      <ImageComponent alt={""} ratio={"1/1"} width={"full"} />
+      <div className="pt-4">
+        <div className="flex justify-between px-4">
+          <div>しまぶーシェフ</div>
```

Comment:
折角モック作ったのであればreturn前に
```
const chef = RecipeAppTeam10[parseInt(id)];
```
みたいな物を作って下記の形にした方が良い気がします！

```suggestion
          <div>{chef.name}</div>
```

Reply:
idに一致したシェフの名前を持ってきたいところですが、直後のdescriptionと同じ方式にしておきます。